Source/WebCore/css/color/CSSUnresolvedColor.cpp

```diff
@@ -45,6 +45,18 @@ bool CSSUnresolvedColor::containsCurrentColor() const
     );
 }
 
+bool CSSUnresolvedColor::containsColorSchemeDependentColor() const
+{
+    return WTF::switchOn(m_value,
+        [&] (const CSSUnresolvedColorMix& unresolved) {
```

Comment:
Don't need the lambda capture-all-by-ref here ?

Reply:
Removed.